### PR TITLE
Python 3.6

### PR DESCRIPTION
**Pull request by galnov**
Adding python 3.6 dependency due to numpy 1.19.0 dropping compatibility with python 3.5.
Also removing intel-tensorflow dependency due to this: https://github.com/pypa/pip/issues/7487#issuecomment-566303039

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -5,14 +5,19 @@ FROM coach-base:master as builder
...
-    && unzip mujoco.zip -d ~/.mujoco \
-    && rm mujoco.zip
+ && wget https://www.roboti.us/download/mujoco200_linux.zip -O mujoco.zip \
+ && unzip mujoco.zip -d ~/.mujoco \
+ && mv /root/.mujoco/mujoco200_linux /root/.mujoco/mujoco200 \
+ && rm mujoco.zip
 ARG MUJOCO_KEY
 ENV MUJOCO_KEY=$MUJOCO_KEY
-ENV LD_LIBRARY_PATH /root/.mujoco/mjpro150/bin:$LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH /root/.mujoco/mujoco200/bin:$LD_LIBRARY_PATH
```
Comment (gal-leibovich) [inline docker/Dockerfile:14]: are we sure we want to move to mujoco 2.0? Ant for instance is not working with Gym. Other than that, I'm also not sure regarding all the existing mujoco presets x all continuous control algorithms, that these will not require any change.

Comment (galnov) [reply docker/Dockerfile:14]: Running the golden tests as part of this PR. Let's see what the results look like.

Comment (gal-leibovich) [reply docker/Dockerfile:14]: Golden tests are not enough... it only tests for inverted pendulum and maybe hopper. If we want to be on the safe side, need to schedule and run everything we have for mujoco in our benchmarks.

Now, if i'm not mistaken, I think that @guyjacob already looked into this in the past and came with a conclusion that mujoco 2.0 levels match the mujoco 1.5 ones, other than `Ant`, but I'm not sure. If that is really the case, then we're good. But if we want to make sure then we might want to schedule runs on the cluster.

**Comment by guyjacob** [comment]
Merged - all environments are built successfully and golden tests passing. Multi-node tests still failing, to be debugged separately.

**Pull request merged**